idaes/generic_models/unit_models/heat_exchanger.py

```diff
@@ -29,7 +29,7 @@
     Block,
     units as pyunits
 )
-
+from pyomo.util.calc_var_value import calculate_variable_from_constraint
```

Comment:
Why was this added in this PR? I don't see it being used in the new code below?

Reply:
added before the initialize method, removed